Features/WatchList/WatchlistTableViewCell.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  WatchlistTableViewCell.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 21.07.22.
+//
+
+import UIKit
+
+class WatchlisttableViewCell: SearchTableViewCell {
```

Comment:
Just a small typo
```suggestion
class WatchlistTableViewCell: SearchTableViewCell {
```
I like the idea of subclassing so you can reuse the component and keep different names. But which one is better? We create another class as a subclass or should we make the first a reusable component. Instead of `SearchTableViewCell` maybe just `MovieListTableViewCell`. This is not a request to change, I jsut want you to think about which one would be better in your opinion, and why

Reply:
I guess the pros of subclassing is that we can modify each screen separately if needed + clear naming:)
On the other hand, could  `MovieListTableCell ` be a template that both of these cells will inherit? I think read about this design pattern.

Reply:
Perfect!